pageserver/src/tenant/mgr.rs

```diff
@@ -1436,7 +1415,92 @@ impl TenantManager {
 
         // The Tenant goes back into the map in Stopping state, it will eventually be removed by DeleteTenantFLow
         slot_guard.revert();
-        result
+        let () = result?;
+        Ok(StatusCode::ACCEPTED)
+    }
+
+    pub(crate) async fn delete_tenant(
```

Comment:
Shouldn't this be placed at above what is now `delete_tenant_attached` so that the code is more readable and one doesn't need to scroll up to see what is the should-be-private function called?

Reply:
Moved this in 6ebbfd0d687fcd140312b3a0a4f8d890be0c9c47